### ISSUE TITLE
Klicktel

### ISSUE THREAD

**Issue by jammerhexe** [nvaccess/nvda#7573]
### Steps to reproduce:
> A list of the steps you take to demonstrate the problem. 
>
> Example:
>
> 1. Open Chrome
> 2. Browse to www.google.com
> 3. Type "Hello"
> 4. Notice an error sound when enter is pressed.

> Please also remember to attach a zip of any files required to reproduce the issue.

### Expected behavior:
> Tell us what should happen.

### Actual behavior:
> Tell us what happens instead.

### System configuration:
NVDA version:
> Example: next-14027,c80e529f

NVDA Installed or portable:

Other information:
> Example: Running in a VM

Windows version:
> Example: Windows 10 Version 1607 Build 14393.1066

Name and version of other software in use when reproducing the issue:

### Other questions:

Does the issue still occur after restarting your PC?

Have you tried any other versions of NVDA?
> Please list them and the result

**Comment by leonardder** [comment]
This issue only has a title. Please edit your initial message with relevant information and a description of your problem/request.

**Comment by Brian1Gaff** [comment]
Klicktel is an adware toolbar for browsers. Something that should be got 
shot of as soon as possible I fear!
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by jammerhexe** [comment]
Sorry if this comes twice. I wrote that I use the very common phone book CD in Germany entitled "Klicktel". NVDA reads the Menues, but unfortunately it didn't read the lists of towns, Names etc. I suppose it only is a minor problem but it would be nice if somebody could have a look.
I uploaded an ISO image to my NAS at:
https://menschomat.goip.de/nas/filelink.lua?id=154cd01141600847
The file is about 800 MB.
